components/AutoComplete.js

```diff
@@ -0,0 +1,37 @@
+const React = require("react");
+const TextInput = require("ink-text-input").default;
+const { Box, Text, useInput } = require("ink");
+
+const AutoComplete = ({valueControl={val, setValue}, baseList, handleSubmit}) => {
+    const [suggestion, setSuggestion] = React.useState([]);
+    const updateSuggestion = (input) => {
+		valueControl.setValue(input);
+		if (input.length > 0) {
+		  const match = baseList.find(w => w.trim().startsWith(input.toLowerCase()));
```

Comment:
We should make sure **both** input and the item from the baseList are lowercase.

```suggestion
          const match = baseList.find(w => w.trim().toLowerCase().startsWith(input.toLowerCase()));
```